src/components/styles/NewTrip.css

```diff
@@ -2,19 +2,195 @@
+  padding-right: 20px;
+  border: solid 2px rgb(255, 255, 255);
+  box-shadow: 2px 2px 5px #181212;
 }
 
 .formHeader,
 .formTravelDates,
 .formTransport,
 .formAccommodation,
 .formSightseeing {
-  border: black solid 1px;
+  border: rgb(255, 255, 255) solid 2px;
   display: flex;
   flex-direction: column;
-  width: 370px;
-  margin: 5px;
+  width: 350px;
+  margin: 3px;
   padding: 5px;
-  background-color: teal;
+  background-color: rgba(37, 57, 73, 0.95);
+  box-shadow: 2px 2px 5px #2e2828;
+  color: white;
+  align-items: center;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  border-radius: 10px;
+}
+
+.formTravelDates {
+  display: flex;
+  flex-direction: row;
+  align-content: center;
+  justify-content: center;
+}
+
+.tripNameForm,
+.inputSightseeingForm {
+  display: flex;
+  flex-direction: column;
   align-items: center;
+  text-transform: uppercase;
+  margin: 0;
+}
+
+.inputAccommodationFormName {
+  display: flex;
+  margin-bottom: 2px;
+}
+
+.inputAccommodationFormName input {
+  margin-left: 5px;
+}
+
+.inputSightseeingForm textarea {
+  margin-top: 2px;
+  width: 320px;
+  height: 100px;
+  font-size: 12px;
+}
+
+.tripNameForm input {
+  width: 190px;
+  height: 30px;
+  padding-left: 4px;
+}
+
+.tripStartForm input,
+.tripEndForm input,
+.inputTranportTimeForm input,
+.inputAccommodationForm input {
+  padding-left: 13px;
+  margin-left: 3px;
+  margin-bottom: 2px;
+}
+
+.checkinCheckout,
+.departureArrival,
+.formTransport {
+  display: flex;
+}
+
+.tripStartForm,
+.tripEndForm,
+.inputAccommodationForm {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+.tripEndForm {
+  margin-left: 2px;
+}
+
+.inputTranportTimeForm {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 20px;
+}
+
+.tripTransportForm {
+  display: flex;
+  margin-bottom: 2px;
+}
+
+.tripTransportForm select {
+  padding-bottom: 2px;
+  margin-left: 5px;
+}
+
+.newTripFormFooter {
+  margin-top: 8px;
+  margin-bottom: 8px;
+  display: flex;
+}
+
+.createNewTripButton,
+.cancelNewTripButton {
+  border: solid 2px rgb(255, 255, 255);
+  background-color: rgba(37, 57, 73, 0.95);
+  color: white;
+  border-radius: 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100px;
+  height: 50px;
+  box-shadow: 2px 2px 5px #181212;
+  font-weight: 700;
+  font-size: 0.9em;
+  cursor: pointer;
+  margin: 0 10px;
+}
+
+.createNewTripButton {
+  background-color: rgb(9, 65, 9);
+}
+
+.cancelNewTrip {
+  text-decoration: none;
+}
+
+.cancelNewTripButton {
+  background-color: rgb(85, 31, 31);
+}
+
+.cancelNewTrip:hover,
+.createNewTripButton:hover {
+  scale: 105%;
+}
+
+@media only screen and (max-width: 350px) {
+  .formHeader,
+  .formTravelDates,
+  .formTransport,
+  .formAccommodation,
+  .formSightseeing {
+    width: 290px;
+    font-size: 0.8em;
+    text-align: center;
+  }
+  .inputTranportTimeForm {
+    margin: 0;
+  }
+  .inputSightseeingForm textarea {
+    width: 260px;
```

Comment:
I expanded the text area and it did this:
<img width="431" alt="Screenshot 2021-05-07 at 09 12 04" src="https://user-images.githubusercontent.com/79857938/117411974-7dd12f80-af14-11eb-9908-6049fc58142e.png">

I don't think anyone else would technically do this with the text area, but I thought I still let you know :)

Reply:
Great! Thanks for checking that Yvonne!